content/blog/2019-01-29-rdhs.md

```diff
@@ -0,0 +1,580 @@
+---
+slug: rdhs
+title: "Interacting with The Demographic and Health Surveys (DHS) Program data"
+categories: blog
+date: '2019-01-29'
+package_version: 0.6.1
+slug: rdhs
+tags:
+  - R
+  - api
+  - Software Peer Review
+  - packages
+  - community
+  - data extraction
+  - survey data
+  - global health
+authors: 
+  - OJ Watson
+topicid: null
+---
+
+There seem to be a lot of ways to write about your R package, and rather than have 
+to decide on what to focus on I thought I'd write a little bit about everything.
+To begin with I thought it best to describe what problem `rdhs` tries to solve, 
+why it was developed and how I came to be involved in this project. I then give a
+really brief overview of what the package can do, before continuing to 
+describe how writing my first proper package and the rOpenSci 
+review process was. Lastly I wanted to share a couple of things that I learnt along 
+the way. These are not very clever or difficult things, 
+but rather things that were difficult to Google, which now I think about it should probably
+be the best metric for a difficult problem. 
+
+### Motivation
+
+#### What is the DHS Program
+
+The [Demographic and Health Survey (DHS) Program](https://www.dhsprogram.com) 
+has collected and disseminated population survey data from 
+over 90 countries for over 30 years. This amounts to over 400 
+surveys that give representative data on health indicators, which in
+many countries provides the key data that mark progress towards targets such as 
+the [Sustainable Development Goals (SDGs)](https://sustainabledevelopment.un.org/sdgs). In addition,
+DHS survey data has been used to inform health policy such as detailing trends in child mortality[^1] 
+and characterising the distribution of malaria control interventions in Africa in order to map the 
```

Comment:
You have used "Malaria" and "malaria". Check whole doc for consistency (maybe capitalization is warranted sometimes but if so, that's not clear)

Reply:
changed to lower case throughout